### PR TITLE
More underscores

### PR DESCRIPTION
**Pull request by jkingdon**
This is the followup for various issues discussed in #3521 

With these changes I can successfully run

```
metamath-knife -v -m --parse-typesetting set.mm
```

using the latest metamath-knife.

### CODE REVIEW

**Review by digama0 — APPROVED** [review]

File: set.mm [hunk 1]
```diff
@@ -109155,7 +109155,7 @@ prove that every set is contained in a weak universe in ZF (see
        of Formalized Mathematics, Vol 1, No 3, May-August 1990.  A Tarski class
        is a set whose existence is ensured by Tarski's axiom A (see ~ ax-groth
        and the equivalent axioms).  Axiom A was first presented in Tarski's
-       article _&Uuml;ber unerreichbare Kardinalzahlen_.  Tarski introduced the
+       article _Ueber unerreichbare Kardinalzahlen_.  Tarski introduced the
```
Comment (avekens) [inline set.mm:109158]: What'Äs the problem here with "_&Uuml;ber"?

Comment (digama0) [reply set.mm:109158]: metamath-exe does not recognize `_` as starting italics unless the next character is alphabetic

**Review by avekens — APPROVED** [review]

File: set.mm [hunk 2]
```diff
@@ -625406,7 +625406,7 @@ family of sets (implicit).  (Contributed by Stefan O'Rear,
        over functions and points in N-dimensional space (which are equivalent
        to functions from an "index set").  Many of the following theorems exist
        to transfer standard facts about functions to elements of function
-       sets._ (Contributed by Stefan O'Rear, 30-Aug-2014.)  (Revised by Stefan
+       sets_.  (Contributed by Stefan O'Rear, 30-Aug-2014.)  (Revised by Stefan
```
Comment (icecream17) [inline set.mm:625409]: would this work? there is a period in the sentence before

Comment (digama0) [reply set.mm:625409]: the ending underscore has to have a letter before it and no letter after it, so yes this should work

**Review by icecream17 — APPROVED** [review]

File: set.mm (hunk 1 above)
Comment (jkingdon) [reply set.mm:109158]: Before:

image

After:

image

File: set.mm (hunk 2 above)
Comment (jkingdon) [reply set.mm:625409]: Before:

image

After:

image

**Pull request merged**